### PR TITLE
Joys and woes of pair programming

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @mauro-oto,

This article is about pair programming, advantages, disadvantages and tips. 

Please check it out.

Thanks!

### CODE REVIEW

File: _posts/2016-05-10-joys-and-woes-of-pair-programming.markdown [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+* **Be communicative**. Guide the navigator through your thought process.
+**Talk** about the alternatives and why you go one way or the other.
+* Keep a **post-it** block right next to you. Write down things that could be
+improved but not during the session.
+* Interrupt only when necessary.
+* If you are the navigator, don't be afraid to grab the keyboard to show a
+**better alternative** to the code that the driver is writing.
+* Put your phone on do not disturb mode.
+
+_[The title of this article was inspired in one of [my favorite chapters](http://home.adelphi.edu/sbloch/class/adages/joy.html) of [The Mythical Man-Month](https://en.wikipedia.org/wiki/The_Mythical_Man-Month) by Fred Brooks]_
```
Comment (mauro-oto) [inline _posts/2016-05-10-joys-and-woes-of-pair-programming.markdown]: @etagwerker I think `was inspired in` should be `was inspired by`.

**Comment by mauro-oto** [comment]
@etagwerker looks good, just had one comment. :+1:

File: _posts/2016-05-10-joys-and-woes-of-pair-programming.markdown (hunk 1 above)
Comment (etagwerker) [inline _posts/2016-05-10-joys-and-woes-of-pair-programming.markdown]: @mauro-oto right, that sounds better 👍

**Comment by etagwerker** [comment]
@mauro-oto done!

**Comment by mauro-oto** [comment]
@etagwerker Thanks! 🍻

**Pull request merged**